x-pack/plugins/licensing/server/licensing_route_handler_context.test.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { BehaviorSubject } from 'rxjs';
+import { ILicense } from './types';
+import { setup } from './__fixtures__/setup';
+import { createRouteHandlerContext } from './licensing_route_handler_context';
+
+describe('licensingRouteHandlerContext', () => {
+  it('provides the initial license value', async () => {
+    const { license } = await setup();
```

Comment:
Could this be satisfied by consuming the `license$` available here?

```js
const { license$, license } = await setup();
const context = createRouteHandlerContext(license$);
const { license: contextResult } = await context({});

expect(contextResult).toBe(license);
```

Reply:
Ah yep, I missed that `license$` was available as part of setup. I'll simplify this!